test/buy-tshirt.spec.ts

```diff
@@ -16,9 +16,10 @@ describe('When open the store page', () => {
       const productListPage: ProductListPage = new ProductListPage();
       const productAddedModalPage: ProductAddedModalPage = new ProductAddedModalPage();
       const summaryStepPage: SummaryStepPage = new SummaryStepPage();
+      const products: string[] = ['Faded Short Sleeve T-shirts'];
 
       await menuContentPage.goToTshirtMenu();
-      await productListPage.addShirtToCart();
+      await productListPage.selectProduct(products[0]);
```

Comment:
It is not needed to create an array, just pass the string: 'Faded Short Sleeve T-shirts'

Reply:
I made it thinking that if there were more products, with the array it would be easy to change the one that we want to buy, I will change it.